pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -2731,9 +2731,11 @@ func newTrackingWorkqueue() *trackingWorkqueue {
 	}
 }
 
-func (t *trackingWorkqueue) Add(item interface{}) {
-	t.queue(item)
-	t.limiter.Add(item)
+func (t *trackingWorkqueue) Add(items ...interface{}) {
+	for _, item := range items {
+		t.queue(item)
```

Comment:
Maybe a simple loop here will not deduplicate multiple items, there still will be a race condition with `Get`.

Reply:
I think a mutex is needed here.

Reply:
This is a stub implementation for the GC controller tests, not used for anything else. It was not previously guarded by a mutex so I don't think one is needed now, it's purely the equivalent of multiple successive calls.

Reply:
OK, this means that not all `Add` implementations should have the ability of deduplication, right?

Reply:
This test does not use multiple adds, the implementation was only changed because interface compliance requires it.